docs/build-transaction-construction.md

```diff
@@ -0,0 +1,208 @@
+---
+id: build-transaction-construction
+title: Transaction Construction and Signing
+sidebar_label: Transaction Construction
+---
+
+This page will discuss the transaction format in Polkadot and how to create, sign, and broadcast
+transactions. Like the other pages in this guide, this page demonstrates some of the available
+tools. Always refer to each tool's documentation when integrating.
+
+## Transaction Format
+
+Polkadot has some basic transaction information that is common to all transactions.
+
+- Address: The SS58-encoded address of the sending account.
```

Comment:
Might be useful to show an example of what this looks like in, say, json or something like that. For example, [here's the cosmos transaction format](https://cosmos.network/rpc/#/Transactions/post_txs) when submitting to their rpc endpoint.